### PR TITLE
Marine Microbes port to CKAN

### PR DESCRIPTION
**Pull request by sztamas**

### CODE REVIEW

File: bpam/apps/common/models.py [hunk 1]
```diff
@@ -191,22 +191,22 @@ class CKANServer(models.Model):
...
     @classmethod
     def primary(cls):
         "Returns the lowest order (primary) mirror"
-        servers = cls.objects.all()
-        if len(servers) > 0:
-            return servers[0]
-        else:
+        first = cls.objects.first()
+        if first is None:
             print("Please set the CKAN servers")
```
Comment (aahunter) [inline bpam/apps/common/models.py]: Any reason we don't use logging here?

Comment (sztamas) [inline bpam/apps/common/models.py]: No reason I can see ...

**Comment by aahunter** [comment]
I've had a look through, looks sensible to me. :+1: 

Worthwhile getting GB to have a look also, so I added him.

**Comment by grahame** [comment]
Looks good, a couple of comments though:

 - the cache table `ckan_big_cache` isn't created when I run up local dev (how should it be created?)
 - I see all the `@cache_` decorators are commented out - should they be turned on before we got to prod?
 - the `CKAN_API_KEY` doesn't seem to be passed through in the dev compose environment

**Pull request merged**